src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -1147,6 +1209,92 @@ internal static void ApplyConfiguration(JObject config, ScriptHostConfiguration
             {
                 scriptConfig.SwaggerEnabled = (bool)swaggerEnabled;
             }
+
+            ApplyLoggerConfig(config, scriptConfig);
+            ApplyApplicationInsightsConfig(config, scriptConfig);
+        }
+
+        internal static void ApplyLoggerConfig(JObject configJson, ScriptHostConfiguration scriptConfig)
```

Comment:
You should have tests for this. For all our other config blocks, we have parsing unit tests (in ScriptHostTests I believe). E.g. ApplyConfiguration_Queues